pkg/server/types.go

```diff
@@ -8,14 +8,15 @@ import (
 )
 
 type Config struct {
-	DisableAgent      bool
-	DisableServiceLB  bool
-	ControlConfig     config.Control
-	Rootless          bool
-	SupervisorPort    int
-	StartupHooks      []cmds.StartupHook
-	LeaderControllers CustomControllers
-	Controllers       CustomControllers
+	DisableAgent       bool
+	DisableServiceLB   bool
+	ControlConfig      config.Control
+	Rootless           bool
+	KlipperLBNamespace string
```

Comment:
One nit - on the K3s side I think we're pretty consistent about referring to it as ServiceLB, even within struct field names. Can we continue with that for this new field?
```suggestion
	DisableServiceLB   bool
	ServiceLBNamespace string
	ControlConfig      config.Control
	Rootless           bool
```
```suggestion
	DisableServiceLB   bool
	ControlConfig      config.Control
	Rootless           bool
	KlipperLBNamespace string
```

Reply:
Renamed